src/processing/vPreProcess/src/vPreProcess.cpp

```diff
@@ -409,6 +427,8 @@ void vPreProcess::run() {
                     } else {
                         qstereo.push_back(v);
                     }
+                    if(corners && IS_CORNER(v._coded_data))
+                        qleft_corners.push_back(v);
```

Comment:
Would it be better to have a separate `qstereo_corners` deque instead of using the left one? That would support proper functionality when both `split_stereo = true` and `combined_stereo=true`.